### PR TITLE
Research redesign repo refactoring

### PR DESCRIPTION
**Pull request by netdesignr**
Hi @domingobishop, 
Can you please review this pull request?

Repo commits:
1. Changed repo root directory to match TNA-base file path
2. Separated action hooks from the WP functions
3. Added functions-global and functions-templates files
4. Moved all custom inline php code into functions and reuse functions were appropriate
5. Added composer.json
6. Installed PHP Unit
7. Unit tested all the functions
8. Added theme screenshot
9. Updated .travis.yml file to check for unit tests
10. Browser tested

Thanks  a lot!

### CODE REVIEW

File: inc/functions/functions-templates.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * Display Authors
+ */
+function displayAuthors()
```
Comment (domjngo) [inline inc/functions/functions-templates.php]: Name convention should follow WP coding standards
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

File: inc/functions/functions-templates.php [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            echo '; ' . $other_authors;
+        }
+    }
+    return false;
+}
+
+/**
+ * @return bool
+ */
+function displaySidebar()
```
Comment (domjngo) [inline inc/functions/functions-templates.php]: Name convention should follow WP coding standards
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

File: inc/functions/functions-templates.php [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            get_sidebar(get_post_meta($post->ID, 'sidebar', true));
+        }
+    }
+    return false;
+}
+
+/**
+ * @return bool
+ */
+function displayDateOfPublication()
```
Comment (domjngo) [inline inc/functions/functions-templates.php]: Name convention should follow WP coding standards
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

File: inc/functions/functions-templates.php [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    if (function_exists('get_post_meta')) {
+        echo date('d F Y', strtotime(get_post_meta($post->ID, 'date_published', true)));
+    }
+    return false;
+}
+
+/**
+ * @return bool
+ */
+function displayKeywordsTaxonomy()
```
Comment (domjngo) [inline inc/functions/functions-templates.php]: Name convention should follow WP coding standards
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

File: inc/functions/functions-templates.php [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        }
+    }
+
+    return false;
+}
+
+/**
+ * @return bool
+ */
+function displayPublishedBy()
```
Comment (domjngo) [inline inc/functions/functions-templates.php]: Name convention should follow WP coding standards
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

File: inc/functions/functions-templates.php [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+        }
+    }
+
+    return false;
+}
+
+/**
+ * @return bool
+ */
+function displayFileUrlFileSize()
```
Comment (domjngo) [inline inc/functions/functions-templates.php]: Name convention should follow WP coding standards
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

**Review by domjngo — COMMENTED** [review]
Review comments

File: inc/functions/functions-templates.php (hunk 1 above)
Comment (netdesignr) [reply inc/functions/functions-templates.php]: Fixed

File: inc/functions/functions-templates.php (hunk 2 above)
Comment (netdesignr) [reply inc/functions/functions-templates.php]: Fixed

File: inc/functions/functions-templates.php (hunk 3 above)
Comment (netdesignr) [reply inc/functions/functions-templates.php]: Fixed

File: inc/functions/functions-templates.php (hunk 4 above)
Comment (netdesignr) [reply inc/functions/functions-templates.php]: Fixed

File: inc/functions/functions-templates.php (hunk 5 above)
Comment (netdesignr) [reply inc/functions/functions-templates.php]: Fixed

File: inc/functions/functions-templates.php (hunk 6 above)
Comment (netdesignr) [reply inc/functions/functions-templates.php]: Fixed

**Review by domjngo — APPROVED** [review]

**Pull request merged**